test/bouture.test.js

```diff
@@ -308,3 +308,19 @@ describe(`Bouture.button({onceClick: e => { state *= 2 }})`, () => {
     expect(state).to.equal(37)
   })
 })
+
+describe(`Bouture('custom-element', Bouture('paper-badge'))`, () => {
```

Comment:
This test only tests that a call to the `Bouture` function returns an object. I think it could be simplified to `Bouture('custom-element')` (no child)